channels/us.m3u

```diff
@@ -548,6 +548,8 @@ https://bonappetit-samsung.amagi.tv/playlist.m3u8
 https://condenast-bonappetit-vizio.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="bonappetit.us" tvg-name="bon appétit" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="Cooking",bon appétit
 https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxbonappetit/CDN/playlist.m3u8
+#EXTINF:-1 tvg-id="BonAppetit.us" tvg-name="Bon Appetit" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="",Bon Appetit
+https://condenast-bonappetit-vizio.amagi.tv/playlist.m3u8
```

Comment:
duplicate of line 548

Reply:
Yeah I wonder why the "Incoming change" on conflict re-added this line.